doppler/README.md

```diff
@@ -0,0 +1,44 @@
+# Agent Check: Doppler
+
+## Overview
+
+[Doppler][1] is a secrets manager designed with both security and developer productivity in mind.
+
+This integration allows Activity Logs from [Doppler][1] to be streamed to Datadog, allowing you to monitor changes to your workplace.
+
+## Setup
+
```

Comment:
Can you please add clear directions a user would take to install the integration? For example, go to the Doppler tile on the integrations page, click connect accounts, you will be redirected to doppler to log in, .. 

This should walk through the full oauth flow a user will take. I recommend testing the oauth flow and creating the steps on the steps you take!

Reply:
Absolutely, I re-tested the full oauth flow and provided more detail here.